lib/WP_Auth0_Options.php

```diff
@@ -129,92 +134,124 @@ public function get_logout_url() {
 		return add_query_arg( 'action', 'logout', site_url( 'wp-login.php', 'login' ) );
 	}
 
+	/**
+	 * Get as a string-separated string and parse to array
```

Comment:
No. That's what the method does internally. What devs should care is "obtain the connections as an array of strings".

Reply:
"string-separated string" 😆